analysis/structure/ip.go

```diff
@@ -3,179 +3,25 @@ package structure
 import (
 	"encoding/binary"
 	"net"
-
-	structureTypes "github.com/activecm/rita/datatypes/structure"
-	"github.com/activecm/rita/resources"
-	mgo "github.com/globalsign/mgo"
-	"github.com/globalsign/mgo/bson"
-	log "github.com/sirupsen/logrus"
 )
 
-//BuildIPv4Collection generates binary representations of the IPv4 addresses in the
-//dataset for use in subnetting, address selection, etc.
-func BuildIPv4Collection(res *resources.Resources) {
-	ssn := res.DB.Session.Copy()
-	defer ssn.Close()
-
-	errorCheck := res.DB.CreateCollection(
-		res.Config.T.Structure.IPv4Table,
-		[]mgo.Index{
-			{Key: []string{"ip"}, Unique: true},
-			{Key: []string{"ipv4_binary"}},
-		},
-	)
-	if errorCheck != nil {
-		res.Log.Error("Failed: ", res.Config.T.Structure.IPv4Table, errorCheck)
-		return
-	}
-
-	buildIPv4Binary(
-		res.DB.GetSelectedDB(),
-		res.Config.T.Structure.HostTable,
-		res.Config.T.Structure.IPv4Table,
-		ssn,
-		res.Log,
-	)
-}
-
-//BuildIPv6Collection generates binary representations of the IPv6 addresses in the
-//dataset for use in subnetting, address selection, etc.
-func BuildIPv6Collection(res *resources.Resources) {
```

Comment:
IPv6 support is not yet ready, but when it is this data will live in the hosts collection.